docker-compose.yml

```diff
@@ -130,6 +130,27 @@ services:
       INFLUXDB_INIT_PWD: "${TTN_DASHBOARD_INFLUXDB_ADMIN_PASSWORD:-!notset}"
       PRE_CREATE_DB: "${TTN_DASHBOARD_INFLUXDB_INITIAL_DATABASE_NAME:-demo}"
 
+  ############# Murugan Changes for influxdb backup
+
+  influxdb-backup:
+    restart: unless-stopped
+    image: muruganchandrasekar/influxdb_backup_1.2.4:latest
+    volumes:
+     # Dircectory for database backup
+      - "${TTN_DASHBOARD_DATA}influx-backup:/tmp/backup"
+     # Connecting influxdb data directory to extra instances
+      - "${TTN_DASHBOARD_DATA}influxdb:/var/lib/influxdb"
+     # dropbox syncing
+#      - "/home/cmurugan/Dropbox/murugan:/tmp/backup"
```

Comment:
Please do not use absolute paths, and don't use your own name for files. Use an environment variable like `"${TTN_DASHBOARD_BACKUP}backup", for two reasons:
- it's more meaningful
- with this syntax, because we omit the "/" before backup, docker will immediately complain and exit, which is what we want. (It will try to use `backup:/tmp/backup` as the volume, which is not valid syntax.)